### PR TITLE
DOCSP-17205 dbase name in prompt

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-17205-dbase-name-in-prompt/reference/customize-prompt/#std-label-customize-the-mongosh-prompt
)

[JIRA](https://jira.mongodb.org/browse/DOCSP-17205)

### CODE REVIEW

File: source/reference/customize-prompt.txt [hunk 1]
```diff
@@ -11,9 +11,9 @@ Customize the :binary:`~bin.mongosh` Prompt
    :backlinks: none
    :depth: 1
 
-The ``prompt`` variable can store strings and JavaScript code to
-customize the :binary:`~bin.mongosh` prompt. Use a function that
-returns a string to display dynamic information in the prompt.
+The :binary:`~bin.mongosh` prompt includes the current database name by
+default. Modify the ``prompt`` variable to display custom strings or to
```
Comment (jason-price-mongodb) [inline source/reference/customize-prompt.txt:15]: "You can modify ..."

File: source/reference/customize-prompt.txt [hunk 2]
```diff
@@ -11,9 +11,9 @@ Customize the :binary:`~bin.mongosh` Prompt
    :backlinks: none
    :depth: 1
 
-The ``prompt`` variable can store strings and JavaScript code to
-customize the :binary:`~bin.mongosh` prompt. Use a function that
-returns a string to display dynamic information in the prompt.
+The :binary:`~bin.mongosh` prompt includes the current database name by
```
Comment (jason-price-mongodb) [inline source/reference/customize-prompt.txt:14]: "By default, the ..."

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM with suggestions. Cheers.

File: source/reference/customize-prompt.txt (hunk 2 above)
Comment (davemungo) [reply source/reference/customize-prompt.txt]: Done

File: source/reference/customize-prompt.txt (hunk 1 above)
Comment (davemungo) [reply source/reference/customize-prompt.txt]: Done

**Review by davemungo — COMMENTED** [review]
@jason-price-mongodb thanks for the feedback, this is ready for another pass.

**Comment by jason-price-mongodb** [comment]
@davemungo LGTM. Cheers.

**Comment by davemungo** [comment]
@addaleax would you take a look at this one when you have a moment?

**Review by addaleax — APPROVED** [review]

**Pull request merged**